src/test/java/org/benetech/servicenet/matching/MatchingAccuracyIntTest.java

```diff
@@ -0,0 +1,186 @@
+        importSmcConnectData();
+        for (Organization org : organizationService.findAll()) {
+            organizationMatchService.createOrUpdateOrganizationMatches(org, null);
+        }
+    }
+
+    @Test
+    public void shouldNotMatchAnyOrganization() {
+        List<Organization> organizations = organizationService.findAll();
+        assertEquals(8, organizations.size());
+        List<OrganizationMatchDTO> matches = organizationMatchService.findAll();
+        assertEquals(0, matches.size());
+    }
+
+    private void importHealthLeadsData() throws IOException {
+        List<String> fileNames = Arrays.asList(
+            ELIGIBILITY, LANGUAGES, LOCATIONS,
+            ORGANIZATIONS, PHONES, PHYSICAL_ADDRESSES,
+            SERVICES_TAXONOMY, TAXONOMY, REQUIRED_DOCUMENTS,
+            SERVICES, SERVICES_AT_LOCATION
+        );
+
+        List<DocumentUpload> uploads = new ArrayList<>();
+        List<String> data = new ArrayList<>();
+        for (String fileName : fileNames) {
+            DocumentUpload upload = new DocumentUpload();
+            upload.setFilename(fileName);
+            uploads.add(upload);
+            data.add(loadCsv(HEALTHLEADS_DIR + fileName + CSV));
+        }
+        MultipleImportData importData = new MultipleImportData(data, uploads, new DataImportReport(), HEALTHLEADS_PROVIDER, true, null);
+        healthleadsDataAdapter.importData(importData);
+    }
+
+    private void importLaacData() throws IOException {
+        String csv = loadCsv(LAAC_DIR + BAD_MATCHES + CSV);
+
+        SingleImportData importData = new SingleImportData(csv, new DataImportReport(), LAAC_PROVIDER, true, null);
+        laacDataAdapter.importData(importData);
+    }
+
+    private void importShelterTechData() throws IOException {
+        String json = AdapterTestsUtils.readResourceAsString(SHELTER_TECH_DIR + BAD_MATCHES + JSON);
+
+        SingleImportData importData = new SingleImportData(json, new DataImportReport(), SHELTER_TECH_PROVIDER, true, null);
+        shelterTechDataAdapter.importData(importData);
+    }
+
+    private void importSmcConnectData() throws IOException {
+        List<String> fileNames = Arrays.asList(
+            ADDRESSES, CONTACTS, HOLIDAY_SCHEDULES,
+            LOCATIONS, MAIL_ADDRESSES, ORGANIZATIONS,
+            PHONES, PROGRAMS, REGULAR_SCHEDULES, SERVICES
+        );
+
+        List<DocumentUpload> uploads = new ArrayList<>();
+        List<String> data = new ArrayList<>();
+        for (String fileName : fileNames) {
+            DocumentUpload upload = new DocumentUpload();
+            upload.setFilename(fileName);
+            uploads.add(upload);
+            data.add(loadCsv(SMC_DIR + fileName + CSV));
+        }
+        MultipleImportData importData = new MultipleImportData(data, uploads, new DataImportReport(), SMC_CONNECT_PROVIDER, true, null);
+        smcConnectDataAdapter.importData(importData);
+    }
+
+    private String loadCsv(String fileName) throws IOException {
```

Comment:
maybe you could move it to the utils class

Reply:
done